Answers.md

```diff
@@ -2,10 +2,21 @@
 
 1.  What is React JS and what problems does it try and solve?
 
+React JS is a JS library that tries to make single page applications fast, scalable, and logical with reusable UI components.
+
 1.  What does it mean to _think_ in react?
 
+Thinking in React means thinking in terms of props, state, and components.
+
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
 
+Class/Stateful components are concerned with holding data for the app and making things work with that data.
+Functional/Presentational components are concerned with how things look and receive data exclusively through props.
+
 1.  Describe state.
 
+State is the object containing all the data about your apps current mode of existence.

```

Comment:
state lives on components, and multiple components can have state, so keep that in mind